build_info.json

```diff
@@ -0,0 +1,63 @@
+{
+  "comment": "Provides build info for use by downstream build rules.",
+  "emboss_py_files": [
```

Comment:
Maybe name this something like `emboss_compiler_files`?  `generated_code_templates` and `prelude.emb` are not py files.

Reply:
Yes, better. I also removed the `emboss_` prefix.